### PR TITLE
Show loading spinner only at startup

### PR DESCRIPTION
**Pull request by eddycharly**
# Changes

This PR ensures we show loading spinner only at startup.

This should fix an issue when web socket disconnects frequently (though the websocket issue should be properly fixed too)

/cc @CarolynMabbott @a-roberts @AlanGreene 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: src/containers/App/App.js [hunk 1]
```diff
@@ -147,9 +147,8 @@ export /* istanbul ignore next */ class App extends Component {
     }
 
     const { loadingConfig } = this.state;
-    const { loadingConfig: prevLoadingConfig } = prevState;
 
-    if (prevLoadingConfig && loadingConfig === false) {
+    if (loadingConfig === false) {
```
Comment (AlanGreene) [inline src/containers/App/App.js:151]: This causes namespaces and extensions to be requested repeatedly.

We could:
- undo these changes and continue using the existing `loadingConfig` in `componentDidUpdate` and `fetchConfig`
- add an additional flag `showLoadingState: true` to the initial state, set it to `false` after `fetchInstallProperties`
- use this to control the loading UI in `render`

Comment (eddycharly) [reply src/containers/App/App.js:151]: Hmm.. let's do that

Comment (eddycharly) [reply src/containers/App/App.js]: done
